### PR TITLE
[GSIP-190] assume +0 vote after 10 days

### PR DESCRIPTION
**Pull request by jodygarnett**
See [GSIP-190](https://github.com/geoserver/geoserver/wiki/GSIP-190) for policy change:

> Assume +0 vote after 10 days

It has been some time since these pages were updated:

* [x] Combines the voting procedure from the gsip and psc pages into an include, and attempt to make them consistent.
* [x] Includs the markdown gsip template as part of the gsip page to save time
* [x] Changes irc meeting to video meeting
* [x] Consistently says 6 months, rather than 2 months, to consider PSC member inactive
* [x] Note OSGeo stuff (annual report, budget, officer, code of conduct)

## Checklist

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [x] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [x] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by travis-ci after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates (screenshots, text)
- [ ] Commits changing the REST API, or any configuration object, should check if the REST API docs (Swagger YAML files and classic documentation) need to be updated.

Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.

### CODE REVIEW

**Comment by aaime** [comment]
This one is good to go right? Proposal is marked as completed: 

* https://github.com/geoserver/geoserver/wiki/Proposals
* https://github.com/geoserver/geoserver/wiki/GSIP-190

**Comment by jodygarnett** [comment]
Yes, just waiting on review.

Review is important as the policy was duplicated in two different sections. Our new change made sentences redundant (like having a quorum of responses to pass a motion). I ended up combing the steps into one file and including it on both the GSP and PSC pages.

The voting.txt file is the one to review.

File: doc/en/developer/source/policies/gsip_template.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# GSIP 1234 - Title
```
Comment (aaime) [inline doc/en/developer/source/policies/gsip_template.txt:1]: We already have a template in the wiki, and the proposals page links to it:
https://github.com/geoserver/geoserver/wiki/GSIP-XXXX

Why duplicate it?

File: doc/en/developer/source/policies/psc.rst [hunk 2]
```diff
@@ -84,49 +84,30 @@ Process
...
-A GSIP is NOT needed for:
+A GSIP proposal is NOT needed for:
 
-  * an improvement that can go in a community module; or
-  * a bug fix that doesn't rework anything substantially
+* an improvement that can go in a community module; or
+* a bug fix that doesn't rework anything substantially
 
-For minor decisions where feedback might be desired, the course of action to take is to consult the development list or raise it in an irc meeting.  The GeoServer Project recognizes that it is run those who are actually doing the work, and thus we want to avoid high overhead for 'getting things done'.
+For minor decisions where feedback might be desired, the course of action to take is to consult the development list or raise it in a video meeting.  The GeoServer Project recognizes that it is run those who are actually doing the work, and thus we want to avoid high overhead for 'getting things done'.
```
Comment (aaime) [inline doc/en/developer/source/policies/psc.rst:108]: This probably needs some amendment. When was this written, the whole PSC was actively participating in meetings, nowadays only a portion takes part in the video calls. We can amend it simply by saying that members not participating to the meeting can follow up after the meeting minutes mail.

**Comment by aaime** [comment]
@jodygarnett some feedback waiting for you in this PR.

File: doc/en/developer/source/policies/gsip_template.txt (hunk 1 above)
Comment (jodygarnett) [reply doc/en/developer/source/policies/gsip_template.txt:1]: Good to remove it from the wiki, as you can not cut and paste right from there. Only reason this template change is when the list of module maintainers changes.

Comment (jodygarnett) [reply doc/en/developer/source/policies/gsip_template.txt:1]: Good to remove it from the wiki, as you can not cut and paste right from there. Only reason this template change is when the list of module maintainers changes.

File: doc/en/developer/source/policies/psc.rst (hunk 2 above)
Comment (jodygarnett) [reply doc/en/developer/source/policies/psc.rst:108]: I think that is already covered by the “developer list” being mentioned.

Comment (jodygarnett) [reply doc/en/developer/source/policies/psc.rst:108]: (In both cases we end up with public communication on the dev list)

Comment (jodygarnett) [reply doc/en/developer/source/policies/psc.rst:108]: (In both cases we end up with public communication on the dev list)

**Pull request merged**